examples/http-tinygo-outbound-http/tinygo-hello/go.sum

```diff
@@ -0,0 +1,2 @@
+github.com/julienschmidt/httprouter v1.3.0 h1:U0609e9tgbseu3rBINet9P48AI/D3oJs4dN7jwJOQ1U=
```

Comment:
Is it correct that we are adding these in this PR?

Reply:
good catch. removed!

Reply:
turns our I did need that go.sum file so I added this back in.